py/nlr.c

```diff
@@ -28,7 +28,9 @@
 
 #if !MICROPY_NLR_SETJMP
```

Comment:
This will still be defined for msvc builds so the new code isn't actually used, or am I missing something?

Reply:
It seems not to be defined under MSVC. The new implementations definitely end up being used. I had to debug them :)